src/models/AppState.js

```diff
@@ -481,6 +481,12 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         fn: function (policy) {
           return policy && policy.rememberDeviceByDefault;
         }
+      },
+      'isUnauthenticatedStatus': {
```

Comment:
Nit: Can we call this "isUnauthenticated"? I.e. similar to how we do other status responses up top (i.e. isMfaChallenge, etc). Also might want to just bump this up closer to those, since it's in the same category.